src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -130,6 +135,46 @@ def update_stats(self):
                         self.mib_oid_list.append(mib_oid)
                         self.mib_oid_to_queue_map[mib_oid] = counter_value
 
+        # Update queue counters for LAG
+        lag_range = sorted(list(self.oid_lag_name_map.keys()))
+        for lag_index in lag_range:
+            lag_oid_list = []
+            lag_oid_to_queue_map = {}
+            # Get counters for each LAG member
+            for lag_member in self.lag_name_if_name_map[self.oid_lag_name_map[lag_index]]:            
+                lag_member_queues = []
+                try:
+                    lag_member_queues = self.port_queue_list_map[mibs.get_index(lag_member)]
+                except KeyError:
+                    # LAG member does not has a queues, continue..
+                    continue
+
+                for queue in lag_member_queues:
+                    # Get queue type and statistics
+                    queue_sai_oid = self.port_queues_map[mibs.queue_key(mibs.get_index(lag_member), queue)]
+                    queue_stat_table_name = mibs.queue_table(queue_sai_oid)
+                    queue_type = self.queue_type_map.get(queue_sai_oid)
+                    queue_stat = self.queue_stat_map.get(queue_stat_table_name, {})
+
+                    # Add supported counters to MIBs list and store counters values
+                    for (counter, counter_type), counter_mib_id in CounterMap.items():
+                        # Only egress queues are supported
```

Comment:
>Only [](start = 26, length = 4)

The same

Reply:
There is appropriate comment already: "# Only egress queues are supported"
Which information do we need to extend?